inference-engine/src/vpu/common/src/utils/ie_helpers.cpp

```diff
@@ -175,4 +175,15 @@ void printTo(DotLabel& lbl, const ie::CNNLayerPtr& ieLayer) {
     subLbl.appendPair("precision", ieLayer->precision.name());
 }
 
+static ie::ICore* _core;
```

Comment:
Am I right that it's thread-safe and work even if several cores are created?

Reply:
As I understand it, this is not related to multithreading. Each Core loads one "Myriad" plugin and `SetCore` method is called only once per instance of Core and plugin.